docs/source/github/github.rst

```diff
@@ -254,6 +254,55 @@ remote origin:
 Again, the pull request will be updated accordingly and CI checks
 will be re-triggered.
 
+Cherry-picking your PR to other branches
+----------------------------------------
+
+After your PR is merged into the master branch, you need to consider whether it should be backported to earlier branches.
+If the content is a new feature designated for the next release, obviously backporting is not appropriate. But if it is a fix or
+update to an existing topic, don't forget to cherry-pick the PR back to earlier branches as needed.
```

Comment:
Mention that when in doubt, consult the maintainer that merged the PR for advice. Both parties should consider the backport and either party can trigger the backport.